src/proto/feature_flag.proto

```diff
@@ -64,4 +64,6 @@ message FeatureFlagList {
   optional bool enable_dns_upstream_query = 5 [default = false];
   // Whether to enable prefix delegation.
   optional bool enable_dhcp6_pd = 6 [default = false];
+  // Whether to enable link metrics manager.
+  optional bool enable_linkmetrics_mgr = 7 [default = false];
```

Comment:
1. Change name to enable_link_metrics_mgr?
2. For default value, if you want to turn it on by default, use default = true.